.github/workflows/go.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     - name: Set up Go 1.x
       uses: actions/setup-go@v3
       with:
-        go-version: ^1.14
+        go-version: ^1.21.7
```

Comment:
should we pin to ^1.21 so we pick up patch releases automatically?

Reply:
Ah thanks, I didn't realize that is what that did :-)